network-api/networkapi/buyersguide/factory.py

```diff
@@ -37,7 +37,7 @@ def get_random_option(options=[]):
 
 
 def get_extended_yes_no_value():
-    return get_random_option(['Yes', 'No', 'NA', 'U'])
+    return get_random_option(['Yes', 'No', 'NA', 'CD'])
```

Comment:
this might be a rebase artifact, and should go away when we rebase `wagtail-localize` on master.